chat-core/src/EmbeddedContent.ts

```diff
@@ -102,7 +111,7 @@ export type FindNearestNeighborsOptions = {
   minScore: number;
 
   /**
-    Atlas Search filter expression.
+    Search filter expression.
```

Comment:
Heh, this does highlight a bias (obviously) to Atlas VS, because the user of the abstract interface will need to know some implementation details in order to provide the right kind of search filter.

Nothing to do now, just something worth keeping in mind for later.

Reply:
good point! 

i think `Record<string, unknown>` is generic enough that it could apply to other vector stores as well, but perhaps we should make this a passable `T` to the method or a constructor func

Reply:
`Record<string, unknown>` is fine but the problem is that the exact shape of that object (think MQL) depends on the actual backing vector store.